source/src/apps/public/analysis/SID_ERMS_Rescore.cc

```diff
@@ -0,0 +1,673 @@
+			if ( breakage_cutoff_ < 0 ) {
+				std::stringstream err_msg;
+				err_msg << "Breakage cutoff must be greater than zero. Value of " << breakage_cutoff_ << " eV given." << std::endl;
+				utility_exit_with_message(err_msg.str());
+			}
+			A_ = 0.002; //default steepness when using breakage cutoff
+		}
+
+		//steepness parameter
+		if ( option[ steepness ].user() ) {
+			A_ = option[ steepness ]();
+			TR.Warning << "Custom steepness of " << A_ << " input. Not recommended." << std::endl;
+		}
+
+		//complex type (nodes and edges)
+		core::Size n_chains_;
+		utility::vector1<char> nodes_;
+		utility::vector1<utility::vector1<std::string>> edges_;
+		if ( option[ complex_type ].user() ) {
+			read_complex_type(n_chains_, nodes_, edges_);
+		} else {
+			std::stringstream err_msg;
+			err_msg << "Must input the complex type using the complex_type option." << std::endl;
+			utility_exit_with_message(err_msg.str());
+		}
+		TR << "Complex type read in" << std::endl;
+
+		//create score function object
+		core::scoring::ScoreFunctionOP sfxn = core::scoring::get_score_function();
+
+		std::ostringstream out;
+
+		out << "Structure" << "\t" << "RMSE" << "\t" << "Ref15_Score" << "\t" << "ERMS Rescore" << std::endl;
+
+		utility::vector1<core::Real> rmse_list;
+		utility::vector1<core::Real> score_list;
+
+		//pose specific input checks
+
+		utility::vector1<std::string> files;
+		utility::vector1<file::FileName> list;
+		if ( option[in::file::l].user() ) {
+			TR << "using -l option " << std::endl;
+			list = basic::options::option[ in::file::l ]();
+			for ( auto const & listfile : list ) {
+				utility::io::izstream pdbs( listfile );
+				std::string fname;
+				while ( pdbs >> fname ) {
+					files.push_back(fname);
+				}
+			}
+			for ( auto const & file : files ) {
+				TR << "creating pose for " << file << std::endl;
+				pose::Pose pose_in;
+				core::import_pose::pose_from_file(pose_in, file, core::import_pose::PDB_file);
+				TR << "created pose for " << file << std::endl;
+				//std::string output = pose_in->pdb_info()->name();
+
+		//set up vector of B values. Length is number of different interface types
+				TR << "edges size is: " << edges_.size() << std::endl;
+				utility::vector1<core::Real> B_(edges_.size(), 0.0);
+				if ( option[ B_vals ].user() ) {
+				//read in B values if given by file input
+					read_B_vals(B_);
+					TR << "B values read in from file" << std::endl;
+					if ( option[ basic::options::OptionKeys::in::file::l ].user() ) {
+					TR.Warning << "PDB file input not used. B values read from file instead. To calculate B from structure, remove the flag (B_vals)." << std::endl;
+					}
+				} else {
+					if ( !option[ basic::options::OptionKeys::in::file::l ].user() ) {
+						std::stringstream err_msg;
+						err_msg << "Must input either file containing B values (using flag B_vals) or PDB file to calculate B values for simulation." << std::endl;
+					utility_exit_with_message(err_msg.str());
+				}
```

Comment:
You need to beautify -- that last curly brace isn't matched with the else clause, despite looking like it is from the indentation.